packages/twenty-server/src/engine/workspace-manager/workspace-migration-runner/utils/custom-table-default-column.util.ts

```diff
@@ -22,4 +26,30 @@ export const customTableDefaultColumns: TableColumnOptions[] = [
     type: 'timestamptz',
     isNullable: true,
   },
+  {
+    name: 'position',
+    type: 'float',
+    isNullable: true,
+  },
+  {
+    name: 'name',
+    type: 'text',
+    isNullable: false,
+    default: "'Untitled'",
+  },
+  {
+    name: 'createdBySource',
+    type: 'enum',
+    enumName: `${tableName}_createdBySource_enum`,
```

Comment:
I'm always confused with that, are we really using camelcase for enum names? @charlesBochet (including the tableName that will be in camelCase as well)

Reply:
yes, everything is camelCase and values should be UPPER_CASE